controlpanel/frontend/jinja2/parameter-list.html

```diff
@@ -0,0 +1,23 @@
+{% from "modal-dialog/macro.html" import modal_dialog %}
+{% from "includes/parameter-list.html" import parameter_list with context %}
+
+{% extends "base.html" %}
+
+{% set page_name = "parameters" %}
+{% set page_title = "Your parameters" %}
+
+{% set access_levels_html %}
+{% include "modals/user_data_access_levels.html" %}
+{% endset %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+
+  {{ parameter_list(object_list) }}
+
+  <p class="govuk-body">
+  <a class="govuk-button" href="{{ url('create-parameter') }}">
+      Create new parameter
```

Comment:
Who has permission to create parameters? You should hide this button from users who don't

Reply:
As far as I can tell all users can create params, is that right?

Reply:
Anyone can create a Parameter. Users can only see their own.